core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -186,9 +190,15 @@ private void validateNoActiveConsumers(final String groupId,
         final List<MemberDescription> members =
             new ArrayList<>(describeResult.describedGroups().get(groupId).get().members());
         if (!members.isEmpty()) {
-            throw new IllegalStateException("Consumer group '" + groupId + "' is still active "
-                    + "and has following members: " + members + ". "
-                    + "Make sure to stop all running application instances before running the reset tool.");
+            if (options.has(forceOption)) {
+                System.out.println("Force deleting all active members in the group: " + groupId);
+                adminClient.removeMembersFromConsumerGroup(groupId, new RemoveMembersFromConsumerGroupOptions()).all();
```

Comment:
Should we check the member removal result here before proceeding? If that call failed, the whole operation should fail with error message containing the result IMHO.

Reply:
Agreed, fixed